cmd/bosun/web/web.go

```diff
@@ -227,6 +228,33 @@ func JSON(h func(miniprofiler.Timer, http.ResponseWriter, *http.Request) (interf
 	})
 }
 
+func Shorten(w http.ResponseWriter, r *http.Request) {
+	u := url.URL{
+		Scheme: "https",
+		Host:   "www.googleapis.com",
+		Path:   "/urlshortener/v1/url",
+	}
+	if schedule.Conf.ShortURLKey != "" {
+		u.RawQuery = "key=" + schedule.Conf.ShortURLKey
```

Comment:
Nope. The docs say: "The API key is safe for embedding in URLs; it doesn't need any encoding."